src/pages/MenuPage.js

```diff
@@ -38,96 +95,66 @@ export class MenuPage extends React.Component {
     database.removeListener(this.databaseListenerId);
   }
 
+  /**
+   * Method will try to match stringified customData object for current store
+   * against keys in MODULES, will return values of keys in MODULES
+   * that match: customData[keyInModules]data === 'true'
+   * @param {string}  customDataAsString  Store custom data as string
+   * @return {array}  result              The maxOrdersPerPeriod of the orderType
+   */
+  getUsedModules = customDataAsString => {
+    if (!customDataAsString || customDataAsString.length === 0) return [];
+
+    const customData = JSON.parse(customDataAsString);
+    const result = [];
+    const checkAndAddToResult = ([moduleName, moduleValues]) => {
+      if (!customData[moduleName]) return;
+      if (!customData[moduleName].data) return;
+      if (customData[moduleName].data === 'true') result.push(moduleValues);
```

Comment:
Seems excessive. Checking if `customData[moduleName]` is falsey, and then checking if it's equal to a specific string.

Could also `.reduce()` and append to the accumulator with an `initialValue = []` rather than pushing

Reply:
Seems odd that the property 'data' would be the string 'true' anyway.

Reply:
removed excessive checking https://github.com/openmsupply/mobile/pull/868/commits/cdc1d9034ff584383b6407ad421f324e2a2bf9f2

I think the syntax of for each and 'external' array is much neater

Reply:
@Chris-Petty that's the format custom data comes in as from mSupply primary